test/powershell/Modules/Microsoft.PowerShell.Utility/Unblock-File.Tests.ps1

```diff
@@ -48,15 +40,15 @@ Describe "Unblock-File" -Tags "CI" {
 
     It "With '-Path': file exist" {
         Unblock-File -Path $testfilepath
-        Test-UnblockFile | Should -BeTrue
+        Test-UnblockFile
```

Comment:
Please revert the change - `Test-UnblockFile` should return True.

Reply:
Fixed.